src/pocketmine/Player.php

```diff
@@ -2413,6 +2413,10 @@ public function handleInventoryTransaction(InventoryTransactionPacket $packet) :
 							$ev->setCancelled();
 						}
 
+						if(!$this->isSprinting() and !$this->isFlying() and $this->fallDistance > 0 and !$this->hasEffect(Effect::BLINDNESS) and !$this->isInsideOfWater()){
+							$ev->setDamage($ev->getDamage() / 2, EntityDamageEvent::MODIFIER_CRITICAL);
```

Comment:
`$ev->getDamage() / 2` - 150%?

Reply:
yeah, but instead of setting the base damage to 150% of the base damage directly, the extra 50% is handled by a different modifier.

Reply:
@Muqsit right, I forgot about this, thanks.